### DISCUSSION TITLE
"For You", ability to exclude suggestions

### DISCUSSION THREAD

**Discussion by jemshit** [community/community#17092] [Other Feature Feedback, Questions, & Ideas]
Hi,

i was just checking "for you" tab, **i like** it because
- i can explore new things that i might like and need to check out
- i can see why particular thing is recommended on my feed (3 dots show it)

There is **one problem** which makes "feed recommendation" **contradictory** with "project star count/visibility/support level":
-  I can see some things on feed are recommended because i starred "some project". But i don't want to see anything related to that project or similar projects on my feed, and there is no way to "exclude" them from my feed -> which forces me to unstar that project
- Here is the **use case**: i used to work with tensorflow/keras/.. libraries and i like them, i star them. I believe number of stars shows how strong library/community is, and even though i don't use them anymore, i want my stars to stay. But since i don't use them now (maybe never), i don't want to follow anything related to them, on my feed. So i need an **option to exclude** them from my feed. But if you don't give me an option, then i have to unstar tensorflow/keras/.. libraries even though i don't want to.

**Comment by siminapasat** [comment]
Thank you for your feedback @jemshit ✨ . We're working on an exclusion / dismissal capability as we speak.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
